source/Ship.cpp

```diff
@@ -765,17 +786,35 @@ void Ship::Save(DataWriter &out) const
 			else
 				out.Write(type, 2. * hardpoint.GetPoint().X(), 2. * hardpoint.GetPoint().Y());
 			double hardpointAngle = hardpoint.GetBaseAngle().Degrees();
-			if(hardpoint.IsParallel() || hardpointAngle)
+			if(hardpoint.IsTurret())
```

Comment:
I'm not to happy with the code for turrets and guns drifting further apart. I hope that we get to more generalized hardpoint types in the future (where some gun slots can also turn somewhat and where some guns might also have a hardpoint sprite and where we also get launchers as a separate hardpoint type).

Reply:
I'll try to generalize the code, because ES doesn't warn redundant parameters...